src/main/java/org/jetbrains/research/anticopypaster/models/PredictionModel.java

```diff
@@ -4,7 +4,4 @@
 
 public abstract class PredictionModel {
     public abstract float predict(FeaturesVector featuresVector);
-    public void logInfo(String filepath){
```

Comment:
This used to be logInfo which any class could in theory use. Since I changed it to be logMetrics and logThresholds, I decided to remove this function and use casting instead, since "debugMetrics" should only be activated with the metrics based plugin